css-values-4/Overview.bs

```diff
@@ -1037,12 +1037,12 @@ URL Processing Model</h4>
 			[=request/credentials mode=] is "same-origin", [=request/use-url-credentials flag=] is
 			set, and whose [=request/referrer=] is |referrer|.
 
-		10. [=/Fetch=] |req|, with <var ignore>taskDestination</var> set to |global|, and <var ignore>processResponseConsumeBody</var>
-			set to the following steps given [=/response=] |res| and Null, failure or byte stream
-			|byteStream|:
+		10. Let |controller| be the result of [=/Fetch|fetching=] |req|, with <var ignore>taskDestination</var>
```

Comment:
No need to make a change here, but just fyi, Bikeshed's conjugation support means you could just write `[=/fetching=]` and it'll link correctly to `/fetch`